client/daemon/peer/piece_manager_mock.go

```diff
@@ -0,0 +1,97 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Add `//go:generate mockgen -destination piece_manager_mock.go -source piece_manager.go -package peer` to piece_manager.go.

Reply:
fixed